tests/test_empty_value_handling.py

```
@@ -0,0 +1,349 @@
+            port=SERVER_PORT,
+            tenant="sys",
+            database=SERVER_DATABASE,
+            user=SERVER_USER,
+            password=SERVER_PASSWORD
+        )
+        
+        # Test connection
+        try:
+            result = client._server._execute("SELECT 1 as test")
+            assert result and result[0].get("test", 1) == 1
+        except Exception as exc:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)